tests/features/homepage.feature

```diff
@@ -188,3 +188,39 @@ Feature: Homepage
+      | Proton Global          | validated | Josse Malhoa | 2019-01-05 10:00 +0100 | Clash of jonnys |
+      | Shiny Shan             | validated | Josse Malhoa | 2018-08-14 17:36 +0200 | Clash of jonnys |
+    And news content:
+      | title                       | headline                 | collection       | topic                                      | state     | publication date     | body                                                                                                                                                                                                                                                 |
+      | Current biodiversity adapt  | Adapt habitats           | Clash of jonnys  | Finance in EU, Supplier exchange, E-health | validated | 2021-04-26T19:09:00Z | Here we combine global maps of human populations and land use over the past 12000 y with current biodiversity data to show that nearly three quarters of nature has long been shaped by histories of human habitation and use by indigenous peoples. |
+      | Environmental tests         | Test practices           | Clash of jonnys  | Employment and Support Allowance           | validated | 2021-01-27T16:12:00Z | With rare exceptions current biodiversity losses are caused not by human conversion or degradation of untouched ecosystems but rather by the appropriation colonization and intensification of use in lands inhabited and used by prior societies.   |
+      | Spatial construction        | Win of wildlands         | Clash of jonnys  | HR, Statistics and Analysis, E-justice     | validated | 2021-02-28T13:15:00Z | Global land use history confirms that empowering the environmental stewardship of Indigenous peoples and local communities will be critical to conserving biodiversity across the planet.                                                            |
+      | Easier transformations      | Ecosystem                | Clash of jonnys  | EU and European Policies                   | validated | 2021-03-29T10:18:00Z | Archaeological evidence shows that by 10000 BCE all societies employed ecologically transformative land use practices including burning hunting species propagation domestication cultivation have left long-term legacies across the biosphere.     |
+    And event content:
+      | title                        | body               | collection      | solution               | state            |
+      | Thick-target collisional     | Evaporation        | Clash of jonnys | Lantern International  | validated        |
+      | Conductive cooling losses    | Single-loop        | Clash of jonnys | Proton Global          | validated        |
+      | Source of SXR plasma supply  | Fast electrons     | Clash of jonnys | Shiny Shan             | validated        |
+      | Stars forming in material    | Colossal material  | Clash of jonnys | Shiny Shan             | validated        |
```

Comment:
We should probably have 12+ of each type, and also mix in some that are in various states in addition to `validation` (eg `draft` or `proposed`).

This way we can check that our filters work correctly and that only published content ends up on the homepage.